### ISSUE TITLE
Discussion: Consider Renaming Package?

### ISSUE THREAD

**Issue by jhnsnc** [jhnsnc/precision-inputs#6]
As I continue to work on these components, I feel that a name like "precision-inputs" or "precision-ui" might be more appropriate (and marketable).

I have already received feedback asking why the controls only work vertically instead of how other jQuery knob components work. I think it would help sell the fact that the knobs you can create with this library are intended for precise control--even with touch input.

I will hold off on this decision for at least a week, but I want to seriously consider a rename before moving out of alpha/beta and into a full v1.0 release.

**Comment by MrTamagotchi** [comment]
I've been giving this a bit of thought and personally I think that a rename is a good idea.

The name _precision-ui_ is really great if you are planning on including more input types in the future. It could include other input types (X-Y-grid, as in Fruity X-Y Controller etc.) but also easily styled regular inputs (range, toggle etc.). I know the latter might not be your focus at the moment, but it would be pretty awesome have this be a one-stop-shop solution for inputs in general.

If you decide to keep it as a stand alone component, I would consider renaming it to something like _precision-input-knob_, as I find _knob-input_ a little "vague sounding", if that makes any sense, haha. 

Bonus: I believe the vertical-control-questions would be easily resolved with a "quick reference" in the FAQ!

**Comment by jhnsnc** [comment]
Thanks for the feedback!

I hope to find some time soon to prepare visually-complete implementations of things like adjustable-range knobs, faders, x/y controllers. I think that the purpose of the base class will become much more clear for people when they can see fully-baked implementations on top of it.

**Comment by jhnsnc** [comment]
I do intend to move forward with changing the name to "precision-inputs".

I'll probably wait until the full version 1.0 release before doing so, but I'm going to start using "precision-inputs" in the code and build output.

**Comment by jhnsnc** [comment]
I went ahead with the rename. This project is now called `precision-inputs`.

The new package on NPM is https://www.npmjs.com/package/precision-inputs and the Git repo now lives at https://github.com/jhnsnc/precision-inputs/

I decided that I should go ahead and make the change rather than continuing to point people to the older package. I recognize that many people had downloaded the previous version, so I did add a deprecated message to `knob-input` pointing to the new package. I decided that making the change now would allow me to promote the project more easily rather than feeling unsure about promoting a package with a name change imminent.
